pkg/terminal/command.go

```diff
@@ -368,6 +369,19 @@ Defines <alias> as an alias to <command> or removes an alias.`},
 	
 If locspec is omitted edit will open the current source file in the editor, otherwise it will open the specified location.`},
 		{aliases: []string{"libraries"}, cmdFn: libraries, helpMsg: `List loaded dynamic libraries`},
+
+		{aliases: []string{"examinemem", "x"}, cmdFn: examineMemoryCmd, helpMsg: `Examine memory: examinemem [-<fmt>] <address>.
+	Fmt is a count followed by a format letter and a size letter.
```

Comment:
I wonder if we could just simplify this. Do we really need to specify byte vs halfword vs word etc...?

I'm leaning towards not giving control over chunk size and simply having flags for `-len` and `-fmt`. Length is always in bytes and maybe defaults to the arch pointer size and fmt defaults to hex.

Reply:
Oh, thx for your guidance. I get something, and need some time to change it.